### ISSUE TITLE
Remote URL replacement

### ISSUE THREAD

**Issue by warrenseine** [sourcegraph/sourcegraph-jetbrains#17]
The [VS Code plugin](https://github.com/sourcegraph/sourcegraph-vscode) provides a useful feature to replace the remote before building the SourceGraph URL. In `settings.json`, it looks like:
```json
  "sourcegraph.remoteUrlReplacements": {
    "git.company.com": "git-web.company.com"
  },
```

I would like to see this setting implemented in the JetBrains plugin as well, ideally in the user-provided `sourcegraph-jetbrains.properties` file.

**Comment by Joelkw** [comment]
@warrenseine apologies for the delay on this! We'll take a look and likely ship this within the next month.

**Comment by warrenseine** [comment]
Awesome, thank you!

**Comment by slimsag** [comment]
FWIW, it is possible to do this by adding a `sourcegraph` Git remote that matches your Sourcegraph instance. e.g.:

```
git remote add sourcegraph https://git-web.company.com/my/repo
```

Then the extension will utilize that code host URL in order to determine the repository URL on Sourcegraph.

**Comment by warrenseine** [comment]
That's good to know, thanks! I still prefer the option to set it in a single place if that's in the roadmap :)

**Comment by muratsu** [comment]
@warrenseine this feature has shipped and is publicly available with v1.2.1!

**Comment by warrenseine** [comment]
Thank you very much!
